lm-api/lm_api/permissions.py

```diff
@@ -6,23 +6,37 @@ class Permissions(str, Enum):
     Describe the permissions that may be used for protecting LM Backend routes.
     """
 
-    LICENSE_SERVER_VIEW = "license-manager:license-server:view"
-    LICENSE_SERVER_EDIT = "license-manager:license-server:edit"
-
-    CLUSTER_VIEW = "license-manager:cluster:view"
-    CLUSTER_EDIT = "license-manager:cluster:edit"
-
-    CONFIG_VIEW = "license-manager:config:view"
-    CONFIG_EDIT = "license-manager:config:edit"
-
-    PRODUCT_VIEW = "license-manager:product:view"
-    PRODUCT_EDIT = "license-manager:product:edit"
-
-    FEATURE_VIEW = "license-manager:feature:view"
-    FEATURE_EDIT = "license-manager:feature:edit"
-
-    JOB_VIEW = "license-manager:job:view"
-    JOB_EDIT = "license-manager:job:edit"
-
-    BOOKING_VIEW = "license-manager:booking:view"
-    BOOKING_EDIT = "license-manager:booking:edit"
+    LICENSE_SERVER_CREATE = "license-manager:license-server:create"
+    LICENSE_SERVER_READ = "license-manager:license-server:read"
+    LICENSE_SERVER_UPDATE = "license-manager:license-server:update"
+    LICENSE_SERVER_DELETE = "license-manager:license-server:delete"
+
+    CLUSTER_CREATE = "license-manager:cluster:create"
+    CLUSTER_READ = "license-manager:cluster:read"
+    CLUSTER_UPDATE = "license-manager:cluster:update"
+    CLUSTER_DELETE = "license-manager:cluster:delete"
```

Comment:
Actually, we don't need these permissions anymore, since we removed the Cluster entity.

Reply:
Addressed in commit ad2f363